app/javascript/legacy/models/map.js

```diff
@@ -12,20 +12,28 @@
  * @param {boolean=true} [params.form_map] - Whether or not it's a form map
  */
 
-import {Filters} from './filter';
+import {
```

Comment:
Lots of formatting changes in this file. @tsanders Do we have/need a style guide/automated linting?

Reply:
@deokpyo Looks like it committed a lot of the webpack generated files.  Can we remove and ignore those?